### PR TITLE
empanada in my mouth

### PR DESCRIPTION
**Pull request by NicholasRostov**

### CODE REVIEW

**Comment by NicholasRostov** [comment]
+1

**Comment by michaelyatco** [comment]
-1

File: app/views/empanadas/show.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Empanadas is love empanadas is life.</h1>
```
Comment (michaelyatco) [inline app/views/empanadas/show.html.erb:1]: Let's get some empanadas - I want the guava and cheese one.

**Comment by samanthamorco** [comment]
+1

**Pull request merged**